server/Dockerfile

```diff
@@ -47,82 +50,68 @@ FROM "${DOCKER_HUB_PROXY}debian:bullseye-slim" as php-build
-    RUN apt-get update; apt-get install -y --no-install-recommends \
-        gcc \
-        git \
-        python3 \
-        python3-dev \
-        python3-pip \
-        python3-setuptools \
-        python3-venv \
-        python3-wheel \
-        libfuzzy-dev \
-        libffi-dev \
-        ca-certificates \
-        curl \
-        && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
 
+    RUN apt-get update; apt-get install -y --no-install-recommends python3-pip git && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
     RUN mkdir /wheels
-
-    WORKDIR /tmp
-
-    RUN git clone --depth 1 https://github.com/CybOXProject/mixbox.git; \
-        cd mixbox || exit; python3 setup.py bdist_wheel -d /wheels; \
-        sed -i 's/-e //g' requirements.txt; pip3 wheel -r requirements.txt --no-cache-dir -w /wheels/
-
-    # install python-maec
-    RUN git clone --depth 1 https://github.com/MAECProject/python-maec.git; \
-        cd python-maec || exit; python3 setup.py bdist_wheel -d /wheels
-
-    # install python-cybox
-    RUN git clone --depth 1 https://github.com/CybOXProject/python-cybox.git; \
-        cd python-cybox || exit; python3 setup.py bdist_wheel -d /wheels; \
-        sed -i 's/-e //g' requirements.txt; pip3 wheel -r requirements.txt --no-cache-dir -w /wheels/
-
-    # install python stix
-    RUN git clone --depth 1 https://github.com/STIXProject/python-stix.git; \
-        cd python-stix || exit; python3 setup.py bdist_wheel -d /wheels; \
-        sed -i 's/-e //g' requirements.txt; pip3 wheel -r requirements.txt --no-cache-dir -w /wheels/
-
-    # install STIX2.0 library to support STIX 2.0 export:
-    # Original Requirements has a bunch of non-required pacakges, force it to only grab wheels for deps from setup.py
-    RUN git clone --depth 1 https://github.com/MISP/cti-python-stix2.git; \
-        cd cti-python-stix2 || exit; python3 setup.py bdist_wheel -d /wheels; \
-        echo "-e ." > requirements.txt; pip3 wheel -r requirements.txt --no-cache-dir -w /wheels/
-
-    # Install the new build tool
-    RUN pip3 install build
-
-    # install PyMISP
-    RUN git clone --depth 1 https://github.com/MISP/PyMISP.git; \
-        cd PyMISP || exit; python3 -m build --wheel --outdir /wheels
-
-    # install pydeep2 (drop-in replacement for pydeep)
-    RUN git clone --depth 1 https://github.com/JakubOnderka/pydeep.git; \
-        cd pydeep || exit; python3 setup.py bdist_wheel -d /wheels
-
-    # Grab other modules we need
-    RUN pip3 wheel --no-cache-dir -w /wheels/ plyara pyzmq redis python-magic lief>=0.13.1
-
-    # Remove extra packages due to incompatible requirements.txt files
-    WORKDIR /wheels
-    RUN find . -name "Sphinx*" | tee /dev/stderr | grep -v "Sphinx-1.5.5" | xargs rm -f
-
+    ARG MISP_TAG
+    ARG MISP_COMMIT
+    # Download MISP using git in the /var/www/ directory. Remove unnecessary items.
+    RUN <<-EOF
+            if [ ! -z "${MISP_COMMIT}" ]; then
```

Comment:
Seems to me that this block has 4 extra leading spaces

Reply:
fixed indentation